src/reducers/reducer.ts

```diff
@@ -0,0 +1,65 @@
+import { State } from '../types/State';
+import { Action } from '../types/Action';
+
+export function stateReducer(state: State, action: Action): State {
+  switch (action.type) {
+    case 'setTodos':
```

Comment:
Would be great to use Enum instead of the hardcoded values

Reply:
done